Podcast/Player.swift

```diff
@@ -3,25 +3,40 @@ import UIKit
 import AVFoundation
 import MediaPlayer
 
+class ListeningDuration {
```

Comment:
Use a struct instead of a class here

Reply:
Actually I shouldn't do this because I end up updating some of ListeningDurations variables. You can't mutate fields in a stuct without adding extra mutable code that is ugly and unnecessary.